### PR TITLE
Rework footer

### PR DESCRIPTION
**Pull request by MariusBluem**
- to have 7 links everywhere
- move imprint to "About Nextcloud" 
- remove AGPLv3 from footer, looks cryptic to me - websites license can be found on privacy-page and GitHub - no need to have this so prominent

@jospoortvliet

### CODE REVIEW

**Comment by jospoortvliet** [comment]
Pls don't remove the 'legal notice' part, are are some sue-happy ppl who might abuse that it isn't using exactly those terms in that place... The rest is perfectly fine ;-)

**Comment by MariusBluem** [comment]
Please review! @LukasReschke @jancborchardt @NinaNC

File: templates/footer.php [hunk 1]
```diff
@@ -14,12 +14,14 @@
                 <li><a href="<?php echo home_url('events') ?>"><?php echo($l->t('Events')) ?></a></li>
                 <li><a href="<?php echo home_url('jobs') ?>"><?php echo($l->t('Jobs')) ?></a></li>
                 <li><a href="<?php echo home_url('/community/code-of-conduct/') ?>"><?php echo($l->t('Code of conduct')) ?></a></li>
-                <li><a href="<?php echo home_url('privacy') ?>"><?php echo($l->t('Privacy and legal')) ?></a></li>
+                <li><a href="<?php echo home_url('privacy') ?>"><?php echo($l->t('Privacy')) ?></a></li>
+                <li><a href="<?php echo home_url('privacy') ?>"><?php echo($l->t('Legal notice')) ?></a></li>
```
Comment (jancborchardt) [inline templates/footer.php]: Same url but different list entries? Why separate them in the first place, I would just keep them together?

File: templates/footer.php [hunk 2]
```diff
@@ -53,7 +56,7 @@
         </div>
     <div class="row">
 		<div class="col-sm-12 text-center">
-		<p  class="revealOnScroll"><a href="https://github.com/nextcloud/nextcloud.com">AGPLv3</a> &copy; </font> <?php echo date('Y'); ?> Nextcloud, <small><a href="/impressum"><?php echo $l->t('Legal Notice') ?></a></small></p>
+		<p  class="revealOnScroll">&copy; </font> <?php echo date('Y'); ?> Nextcloud GmbH</p>
```
Comment (jancborchardt) [inline templates/footer.php]: Why remove the website source code link? It’s a good way to let people know we’re fully open source. Maybe not call the link »AGPLv3« but »website source« or so.

Comment (MariusBluem) [reply templates/footer.php]: The websites code can be found on GitHub. You can find this information on the legal&privacy page and on the "Code on GitHub"-link in the footer.

File: templates/footer.php (hunk 1 above)
Comment (MariusBluem) [reply templates/footer.php]: Fixed.

**Pull request merged**